NEWS.rst

```diff
@@ -3,6 +3,109 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.1.0 (2022-02-03)
+===========================
+
+Features
+--------
+
+- Python 3.10 is now a supported platform (#10224)
+- Type annotations have been added to the twisted.python.fakepwd module. (#10287)
+
+
+Bugfixes
+--------
+
+- twisted.internet.defer.inlineCallbacks has an improved type annotation, to avoid typing errors when it is used on a function which returns a non-None result. (#10231)
+- ``twisted.internet.base.DelayedCall.__repr__`` and ``twisted.internet.task.LoopingCall.__repr__`` had the changes from #10155 reverted to accept non-function callables.  (#10235)
+- Revert the removal of .whl building that was done as part of #10177. (#10236)
```

Comment:
Does this need to be mentioned at all? I don't see how it is user-visible; it's a CI change.

Reply:
this is more a reminder for twisted devs to know that wheels are generated via tox and that we run the tests based on the wheels.

It is true that this is not really note-worthiness ... but the release notes are not that big :)